### PR TITLE
starkmatt/c03 aws01

### PR DESCRIPTION
**Pull request by starkmatt**
# Description

Exercise C03-AWS-E01

Student Name: Matt G

Ran into some issues initially with Public IP address not select correctly when creating instance config, and not using the correct AMI Build.
After speaking with the class, I found out the issue and all sorted!

## Type of change

- [x] Exercise Submission (one exercise per PR)
  - [ ] This PR is just for **one** exercise of a class
  - [ ] PR name follows the pattern `<github-username>/<exercise-number>`
  - [ ] Added exercise files inside folder `/classes/<class- number>/exercises/<exercise-number>/<github-username>/`
	- [ ] Review [exercise submission steps](./README.md#Exercises)
- [ ] Class content (instructors/admins)
- [ ] Documentation update
- [ ] Repo management

### CODE REVIEW

**Comment by starkmatt** [comment]
Apologies for the multiple files attached.
I didn't push correctly!

Would I be able to remove the files from this pull request without any issues ?

**Comment by caiocezart** [comment]
You will have to rebase your changes from master branch. In other words, you started your work from the branch that you submitted c03-docker02 exercise and that carried the changes across.

Either rebase your branch, or wait until the PR #231 gets merged to master, that might help clear these changes here.

**Comment by starkmatt** [comment]
> You will have to rebase your changes from master branch. In other words, you started your work from the branch that you submitted c03-docker02 exercise and that carried the changes across.
> 
> Either rebase your branch, or wait until the PR #231 gets merged to master, that might help clear these changes here.

PR# 231 Has been merged!
Resolved the conflicts, should be all good now!

File: classes/03class/exercises/c03-docker02/starkmatt/c03-docker02.txt [hunk 1]
```diff
@@ -7,10 +7,15 @@
```
Comment (denstorti) [inline classes/03class/exercises/c03-docker02/starkmatt/c03-docker02.txt]: This PR should be about `c03-aws01` only.

Can you remove `c03-docker02.txt`? 
Thanks!

File: classes/03class/exercises/c03-aws01/starkmatt/c03-aws01.txt [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+Instance #2  13.236.84.131
+
+what this line is doing? curl https://169.254.169.254/latest/meta-data/hostname > index.html
+
+Looks like thsi line is calling the metadata of the EC2 Instance and writing it to the index.html.
+The user data then after moves it to /var/www/html and places the file in there so when you navigate to the IP address via the browser it'll
+the curl command results in the html document.
+
+I had some issues with port 80 as the Linix image I used for the build was an older AMI and didn't support systemctl.
+After changing the instance to AMI 2 the build worked with no issues!
```
Comment (denstorti) [inline classes/03class/exercises/c03-aws01/starkmatt/c03-aws01.txt]: Very well explained! Congrats!

Keep going 😄

Comment (starkmatt) [reply classes/03class/exercises/c03-aws01/starkmatt/c03-aws01.txt]: Thank you Denis 😄 !

**Pull request merged**